bin/deployment_notification.js

```diff
@@ -44,6 +44,9 @@ switch (commit.branch) {
   case 'dev':
     build.url = 'https://wire-webapp-dev.zinfra.io/auth/?env=prod#login';
     break;
+  case 'edge':
+    build.url = 'https://wire-webapp-edge.zinfra.io/auth/?env=staging#login';
```

Comment:
This should point to `https://wire-webapp-edge.zinfra.io/login/?env=staging#login` instead for convenience - with the others being adjusted.